### PR TITLE
Support Water sensor

### PR DESCRIPTION
**Pull request by roflmao**

### CODE REVIEW

File: pydeconz/sensor.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 SWITCH = ['ZHASwitch', 'ZGPSwitch', 'CLIPSwitch']
 TEMPERATURE = ['ZHATemperature', 'CLIPTemperature']
```
Comment (Kane610) [inline pydeconz/sensor.py]: You need to define WATER as well. Add according to alphabetical order.

File: pydeconz/sensor.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
...
 
-DECONZ_BINARY_SENSOR = FIRE + OPENCLOSE + PRESENCE
+DECONZ_BINARY_SENSOR = WATER + FIRE + OPENCLOSE + PRESENCE
```
Comment (Kane610) [inline pydeconz/sensor.py]: Move this according to alphabetical order.

File: pydeconz/sensor.py [hunk 3]
```diff
@@ -89,6 +89,34 @@ def sensor_unit(self):
         return self._sensor_unit
 
 
+class Water(DeconzSensor):
```
Comment (Kane610) [inline pydeconz/sensor.py]: Move this according to alphabetical order.

File: pydeconz/sensor.py [hunk 4]
```diff
@@ -89,6 +89,34 @@ def sensor_unit(self):
...
+        """Main state of sensor."""
+        return self.is_tripped
+
+    @property
+    def is_tripped(self):
+        """Sensor is tripped."""
+        return self.water
+
+    @property
+    def fire(self):
```
Comment (Kane610) [inline pydeconz/sensor.py]: fire is not water ;)

File: pydeconz/sensor.py [hunk 5]
```diff
@@ -89,6 +89,34 @@ def sensor_unit(self):
...
 
+class Water(DeconzSensor):
+    """Water sensor.
+
+    State parameter is a boolean named 'water'.
+    """
+
+    def __init__(self, device_id, device):
+        """Initialize Water sensor."""
+        self._open = device['state'].get('water')
```
Comment (Kane610) [inline pydeconz/sensor.py]: copy paste issue I just saw, this should be called _water not _open.

File: pydeconz/sensor.py [hunk 6]
```diff
@@ -327,7 +355,9 @@ def temperature(self):
 
 def create_sensor(sensor_id, sensor):
     """Simplify creating sensor by not needing to know type."""
-    if sensor['type'] in FIRE:
+    if sensor['type'] in WATER:
```
Comment (Kane610) [inline pydeconz/sensor.py]: Move this according to alphabetical order.

**Comment by Kane610** [comment]
Im merging and fixing this locally to make sure that they will take the fix for the fire detector as well.

**Comment by roflmao** [comment]
@Kane610 Thanks!

**Pull request merged**